second_test/second_test.spec.js

```diff
@@ -0,0 +1,37 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test('Perform login', async ({ page }) => {
+	await page.goto('https://www.skroutz.gr/');
+	//await page.getByRole('link', { name: 'Σύνδεση / Εγγραφή' }).click();
+	await page.click('a.icon');
+	await page.getByPlaceholder('Email').click();
```

Comment:
A more solid choice would be the field's ID such as `email-input-user[username]`